setup.py

```diff
@@ -18,10 +18,21 @@
                 "That indicates this copy of the source code is incomplete.")
         sys.exit(2)
 
+longdesc = 'Ansible is a radically simple IT automation system. It handles configuration-management, application deployment, cloud provisioning, ad-hoc ' \
+    'task-execution, and multinode orchestration - including trivializing things like zero downtime rolling updates with load balancers.\n' \
+    '\n' \
+    'Read the documentation and more at https://ansible.com/'
+
+
+if os.path.exists('README.rst'):
+    with open('README.rst', 'r+') as readme_file:
```

Comment:
This should not be opened with "+" (as that allows writing the file).  I can't remember off the top of my head whether python2/python3 setuptools expects text strings or byte strings for longdesc.  My guess is text strings.  Since there isn't any non-ascii characted in README.rst I think we can ignore that for now and just use native strings but we might need to figure it out in the future.